usvm-ts/src/main/kotlin/org/usvm/machine/expr/CallApproximations.kt

```diff
@@ -773,19 +834,26 @@ private fun TsExprResolver.handleArrayReverse(
         val reversedLengthLValue = mkArrayLengthLValue(reversedArray, arrayType)
         memory.write(reversedLengthLValue, length, guard = trueExpr)
 
-        // Copy the entire content symbolically - the actual reversal semantics
-        // would require more sophisticated constraint solving
-        memory.memcpy(
-            srcRef = array,
-            dstRef = reversedArray,
-            type = arrayType,
-            elementSort = elementSort,
-            fromSrc = 0.toBv().asExpr(sizeSort),
-            fromDst = 0.toBv().asExpr(sizeSort),
-            length = length,
+        // Initialize the reversed array with symbolic values appropriate for the element sort
```

Comment:
Add a log that we encountered unsupported instruction, therefore it is a point for path divergence